### PR TITLE
Add doc_url, run test on Windows, remove accidental maintainer

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
Travis stalled out: close/reopen to redo.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,15 +34,16 @@ test:
...
     dev_url: http://github.com/inducer/{{ name }}
+    doc_url: https://documen.tician.de/{{ name }}
     license: MIT
     license_file: LICENSE
     summary: A package for symbolic computation
 
 extra:
     recipe-maintainers:
         - inducer
-        - ocefpaf
```
Comment (ocefpaf) [inline recipe/meta.yaml:48]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,6 +11,10 @@ source:
     url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: {{ sha256 }}
 
+    # Delete in the next version past 2016.2.3.
+    patches:
+    - geo-alg-integrality-test.patch
+
 build:
     number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Build number bump (given the patch)?

Comment (inducer) [reply recipe/meta.yaml]: Yep, good catch. Done.

**Pull request merged**